### PR TITLE
Update max content width options

### PR DESCRIPTION
**Pull request by ilpav12**
Updated the maximum content width options and the corresponding Filament's documentation. Added `Zero`, `None`, `MinContent`, `MaxContent`, `FitContent` to the MaxWidth Enum options and made adjustments in the system to match these changes. This enhancement provides more width options to accommodate varying screen sizes and improve the user interface layout on larger screens.

- [x] Changes have been thoroughly tested to not break existing functionality.
- [x] New functionality has been documented or existing documentation has been updated to reflect changes.
- [x] Visual changes are explained in the PR description using a screenshot/recording of before and after.

### CODE REVIEW

**Comment by ilpav12** [comment]
I am not sure if moving the tailwind classes strings from the blade to the Enum will stop tailwind from the generation of those in the css.

**Comment by danharrin** [comment]
They should not be in the enum for that reason

**Comment by danharrin** [comment]
What is the use of the extra options that you cant accomplish with existing options?

**Comment by ilpav12** [comment]
`max-w-fit` for example it is not available on the Filament css so I wanted to provide all the possible max width options available out of the box.

**Comment by danharrin** [comment]
Right, but what is that actually useful for in this context?

**Comment by ilpav12** [comment]
I have some tables that have a width between 7xl and full and with max-w-fit I can get the perfect width even if it changes slightly between pages. I don't know if this replies to your question

**Comment by ilpav12** [comment]
I moved back the classes in the blade file. Now the only difference is the added options.

**Comment by danharrin** [comment]
What would be the point of Zero and None?

**Comment by ilpav12** [comment]
In this case I don't think there is a useful use case but I wanted to add all the options for completeness. Before in the Enum there was an additional case `Screen` that doesn't exist in the tailwind classes.

**Comment by danharrin** [comment]
Can we remove the ones that have no practical use please 😁

**Comment by ilpav12** [comment]
Done: [Removed Zero and None options](https://github.com/filamentphp/filament/pull/9913/commits/2f39ebda5218a76a52ea2ceb61a51f23941766d0)

File: packages/panels/docs/09-configuration.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ public function panel(Panel $panel): Panel
...
+By default, Filament will restrict the width of the content on the page, so it doesn't become too wide on large screens. To change this, you may override the `getMaxContentWidth()` method. Options correspond to [Tailwind's max-width scale](https://tailwindcss.com/docs/max-width). The options are `ExtraSmall`, `Small`, `Medium`, `Large`, `ExtraLarge`, `TwoExtraLarge`, `ThreeExtraLarge`, `FourExtraLarge`, `FiveExtraLarge`, `SixExtraLarge`, `SevenExtraLarge`, `Full`, `MinContent`, `MaxContent`, `FitContent`,  `Prose`, `ScreenSmall`, `ScreenMedium`, `ScreenLarge`, `ScreenExtraLarge` and `ScreenTwoExtraLarge`. The default is `SevenExtraLarge`:
```
Comment (danharrin) [inline packages/panels/docs/09-configuration.md]:
```suggestion
By default, Filament will restrict the width of the content on the page, so it doesn't become too wide on large screens. To change this, you may use the `maxContentWidth()` method. Options correspond to [Tailwind's max-width scale](https://tailwindcss.com/docs/max-width). The options are `ExtraSmall`, `Small`, `Medium`, `Large`, `ExtraLarge`, `TwoExtraLarge`, `ThreeExtraLarge`, `FourExtraLarge`, `FiveExtraLarge`, `SixExtraLarge`, `SevenExtraLarge`, `Full`, `MinContent`, `MaxContent`, `FitContent`,  `Prose`, `ScreenSmall`, `ScreenMedium`, `ScreenLarge`, `ScreenExtraLarge` and `ScreenTwoExtraLarge`. The default is `SevenExtraLarge`:
```

File: packages/support/src/Enums/MaxWidth.php [hunk 2]
```diff
@@ -45,4 +45,6 @@ enum MaxWidth
     case ScreenExtraLarge;
 
     case ScreenTwoExtraLarge;
+
+    case Screen;
```
Comment (ilpav12) [inline packages/support/src/Enums/MaxWidth.php:49]: Why is screen back? Have I missed a spot where it is used because it is not a valid tailwind class

Comment (danharrin) [reply packages/support/src/Enums/MaxWidth.php:49]: It's used in the modal to apply `fixed inset-0`. Please search the codebase before removing stuff

**Pull request merged**